cishouseholds/merge.py

```diff
@@ -40,10 +47,22 @@ def union_multiple_tables(tables: List[DataFrame]):
         list of objects representing the respective input tables
     """
     merged_df = tables[0]
-
     for i, table_n in enumerate(tables[1:]):
-        merged_df, dfn = prepare_for_union(merged_df, table_n)
-        merged_df = merged_df.unionByName(dfn)
```

Comment:
Might be worth changing this into its own function? The original `union_multiple_tables` was used elsewhere (maybe in the merge) and would now be broken because this version hard-codes the table names.